src/cli/.azuredevops/templates/publish-package.yml

```diff
@@ -9,14 +9,13 @@ parameters:
 steps:
 - task: FlutterCommand@0
   displayName: 'Flutter Pub Publish Dry Run'
-  condition: ${{ parameters.dryRun }}
```

Comment:
Not sure why we would remove this parameter usage?

Reply:
Yeah it should be needed since we don't want to actually build it for real in a PR check.

Reply:
This is for the dry run no? The rationale behind removing it is that we want to perform a dry run always, regardless of whether we're deploying or not. For the actual publishing step the parameter is taking into consideration (it won't execute if `parameters.dryRun` is true

Reply:
Yes basically a dry run is a simulation so if we just want some validation during the PR we want dry run, if this is an actual release run, don't run dry run, just build the package, it will do that same validation but create our binaries.

Reply:
Added back